test/bin/common.sh

```diff
@@ -87,6 +87,20 @@ SCENARIO_INFO_DIR="${SCENARIO_INFO_DIR:-${IMAGEDIR}/scenario-info}"
 # The CI system will override this depending on the job its running.
 SCENARIO_SOURCES="${SCENARIO_SOURCES:-${TESTDIR}/scenarios}"
 
+# Directory where all the scenarios will be copied for execution
+#
+# shellcheck disable=SC2034  # used elsewhere
+SCENARIOS_TO_RUN="${OUTPUTDIR}/scenarios-to-run"
+
+prepare_scenario_sources() {
+    rm -rf "${SCENARIOS_TO_RUN}"
+    mkdir -p "${SCENARIOS_TO_RUN}"
+    cp "${SCENARIO_SOURCES}"/*.sh "${SCENARIOS_TO_RUN}"/
+    if [ "${JOB_TYPE+x}" = "x" ] && [ "${JOB_TYPE}" = "presubmit" ]; then
+        rm -f "${SCENARIOS_TO_RUN}/*cncf-conformance.sh"
```

Comment:
I'm not sure we should be deleting files from the repo. We're running these scripts in the devenv, so such a deletion may cause undesired effects.